Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:6
+EXPOSE 3000
+
+RUN apt-get update && apt-get install -y apt-utils
+RUN apt-get install -y libarchive-dev libtinyxml-dev && useradd --user-group --create-home --shell /bin/false app
+
+ENV HOME=/home/app
+
+COPY package.json $HOME/ck-ospell/
```

Comment:
Since you are building an image for a specific version of the app, it might make sense to also copy an `npm-shrinkwrap.json`. Which I have not yet added – but can easily create one yourself using `npm shrinkwrap` (maybe with `--dev`).